rollup.config.js

```diff
@@ -1,8 +1,16 @@
 import nodeResolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
 import babel from 'rollup-plugin-babel';
+import uglify from 'rollup-plugin-uglify';
```

Comment:
May we use `rollup-plugin-babel-minify` instead? I got better outputs with it in Popper.js

Reply:
uglify 37.75 kb
babel-minify 38.15 kb

Reply:
You may need to tweak the default config a bit:

Reply:
I did it. Nothing is changed.

Reply:
Cool thanks!